legal-api/tests/unit/models/test_legal_entity.py

```diff
@@ -548,7 +554,15 @@ def test_amalgamated_into_business_json(session, test_name, existing_business_st
         amalgamation.amalgamating_businesses.append(amalgamating_business)
 
         business.amalgamation.append(amalgamation)
-        business.save()
+        db.session.add(business)
+        existing_business.state_filing_id = filing.id
+        existing_business.state = existing_business_state
+        db.session.add(existing_business)
+        db.session.commit()
+
+        filing.transaction_id = transaction.id
+        filing.business_id = business.id
```

Comment:
i think this should be `filing.legal_entity_id = business.id`

Reply:
found other instances of this, will be fixing all of them